packages/modelviewer.dev/data/examples.json

```diff
@@ -31,19 +31,19 @@
     "examples": [
       {
         "htmlId": "demo-container-1",
-        "name": "Autoplay animation"
+        "name": "Autoplay"
       },
       {
         "htmlId": "demo-container-2",
-        "name": "Select specific"
+        "name": "Selecting Animations"
```

Comment:
The lazy loading examples links are broken.

Reply:
any links in particular? I'm not sure I see that

Reply:
The bullets that say "See our section on lazy loading for examples."

Reply:
yeah